### ISSUE TITLE
CH EMED Organization (Quentin Ligier, ahdis)

### ISSUE THREAD

**Issue by ig-feedback** [hl7ch/ch-emed#294]
ch.fhir.ig.ch-emed#5.0.0-ballot /StructureDefinition-ch-emed-organization.html

The organization address is required, most probably because it was required in the IPAG report (https://docs.google.com/spreadsheets/d/1Ui3NGFE2I8yiOlHELk-B0Pke2l9-Jbe5BTeYOnS8-uE/edit?gid=0#gid=0)
In the EPR and the HPD, the address is not required.
It's an issue for the PMP Portal, which creates Practitioner/PractitionerRole/Organization resources depending on the HPD content. It is unable to generate a valid document when the organization has no address in the HPD.

Quentin Ligier, ahdis

**Comment by qligier** [comment]
https://fhir.ch/ig/ch-emed/StructureDefinition-ch-emed-organization.html

**Comment by ziegm** [comment]
yes, the assumption of the origin of this requirement is correct.

@qligier will attend at the meeting on the 4.8.24 to discuss

**Comment by dvribeira** [comment]
Not sure how relevant this might be, but this is also mandatory on the eMediplan specs. Hence misaligning this now might be also a problem for the eMedication service and other systems that might have to support export to eMediplan.

**Comment by qligier** [comment]
Thanks @dvribeira, that is quite relevant! In CHMED16AF, it is not constrained, I'm not sure if both formats are supposed to be compatible.

**Comment by dvribeira** [comment]
For what I have seen CHMED16A (eMediplan proper, not the FHIR _translation_) neither the practitioner nor the organization are modelled, the model in general being quite simpler than CHMED23A:
![imagem](https://github.com/user-attachments/assets/c801019e-8f4a-4830-8180-e06e4018611e)

The FHIR translation from CHMED23A is yet to be published, not sure if there is an updated ETA on this. An updated version of the print format should also be published at some point this year I believe, since it has been spun off the main spec document.

**Comment by dvribeira** [comment]
Given that this is a _new_ constraint on the eMediplan specs and that maybe to some degree the new model has been updated to better accommodate translations to/from the FHIR version, maybe on the eMediplan side they would agree on relaxing the constraint as well...

**Comment by qligier** [comment]
PJ OE QL SK
IPAG defines the optionality of the organization address.
It is required in the eMediplan.
It is optional in the vaccination format, and more generally in the EPR.

The issue is kept open to gather feedback.

**Comment by oliveregger** [comment]
CHMED16A Revision 2 introduces also Organization for ePrescription: https://github.com/ig-emediplan/specification/pull/1/files
